app/dashboard/templates/shared/accept_work_banner.html

```diff
@@ -0,0 +1,33 @@
+{% comment %}
+  Copyright (C) 2018 Gitcoin Core
+
+  This program is free software: you can redistribute it and/or modify
+  it under the terms of the GNU Affero General Public License as published
+  by the Free Software Foundation, either version 3 of the License, or
+  (at your option) any later version.
+
+  This program is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+  GNU Affero General Public License for more details.
+
+  You should have received a copy of the GNU Affero General Public License
+  along with this program. If not, see <http://www.gnu.org/licenses/>.
+{% endcomment %}
+{% load i18n static %}
+<div id="no_issue_error" class="submit-notification container-fluid" style="display: block; background-color: #fe006a;">
+  <div class="row align-items-center px-4 pt-2 text-center text-lg-left">
+    <div class="col-lg-10 col-sm-12">
+      <h3>{% trans "Accepting a Submission is Exciting" %}</h3>
+      <p>
+        <ol>
+          <li>{% trans "Make sure the submitter has completed the requirements to the specifications of your bounty." %}</li>
+          <li>{% trans "Feedbacks and gratitude are always nice ways to continue a relationship." %}</li>
+        </ol>
+      </p>
+    </div>
+    <div class="col-lg-2 col-sm-12">
+      <img src="{% static "v2/images/Box.svg" %}">
```

Comment:
rename to `box.svg` cause I guess that's what we've been doing soo far in this repo

Reply:
Well, there are mixed namings, so I didn't refactor. If you want, I will.